crates/ruff_python_formatter/tests/snapshots/format@expression__slice.py.snap

```diff
@@ -155,7 +167,8 @@ d1 = "d"[  # comment
     :
 ]
 d2 = "d"[  # comment
-    1:
+    1
+    :
```

Comment:
Hmm, this isn't ideal. Do we need to group the line break with the left side so that it only renders if the left expands (or the `:` and the right side don't fit on the line)